spec/support/capybara.rb

```diff
@@ -1,8 +1,12 @@
-require "capybara/poltergeist"
+require "selenium/webdriver"
 
-Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, js_errors: true, inspector: true)
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
 end
 
-Capybara.default_driver    = :rack_test
-Capybara.javascript_driver = :poltergeist
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(chromeOptions: { args: %w[headless] })
```

Comment:
It might be worth putting in a comment to explain some other values that can be useful in `chromeOptions[:args]`. I'm not sure if/why they are necessary, but our project includes `%[headless disable-cpu no-sandbox]`. Also, in some cases we've had to explicitly specify the dimensions of the browser. E.g. `"window-size=2160,3840"` as an additional value for `:args`.